openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -46,6 +49,11 @@
     'hr_expense_analytic_plans': 'hr_expense_analytic_distribution',
     # OCA/project
     'project_work_time_control': 'project_timesheet_time_control',
+    # OCA/manufacture-reporting
+    'mrp_bom_structure_xls': 'mrp_bom_structure_xlsx',
+    'mrp_bom_structure_xls_level_1': 'mrp_bom_structure_xlsx_level_1',
+    # OCA/reporting-engine
+    'report_xls': 'report_xlsx',
```

Comment:
OK, I thought it was ported (you did the PR), then let's remove it definitively report_xls in OCA in both v9 and v10.